templates/Makefile

```diff
@@ -1,36 +1,34 @@
+PLATFORM = $(shell uname -m)
+
+ifneq ($(PLATFORM),aarch64)
 ifeq ($(origin AS),default)
 AS = aarch64-linux-gnu-as
 endif
 ifeq ($(origin CC),default)
 CC = aarch64-linux-gnu-gcc
 endif
+endif
 
-CFLAGS ?= -g -Wall -Wextra -pedantic -Werror
-LDFLAGS =
-
-ALL_LDFLAGS = -pie -Wl,--fatal-warnings
-
-ALL_CFLAGS = -std=c99 -fPIE $(CFLAGS)
-ALL_LDFLAGS += $(LDFLAGS)
+CFLAGS ?= -g -Wall -Wextra -pedantic -Werror -std=c99 -fPIE 
+LDFLAGS ?= -pie -Wl,--fatal-warnings
 
 C_OBJS = $(patsubst %.c,%.o,$(wildcard *.c))
 AS_OBJS = $(patsubst %.s,%.o,$(wildcard *.s))
 ALL_OBJS = $(filter-out example.o,$(C_OBJS) $(AS_OBJS) vendor/unity.o)
 
 CC_CMD = $(CC) $(ALL_CFLAGS) -c -o $@ $<
 
-PLATFORM = $(shell uname -m)
 ifeq ($(PLATFORM),aarch64)
-	MAYBE_QEMU =
+	MAYBE_QEMU ?=
 else
-	MAYBE_QEMU = qemu-aarch64 -L /usr/aarch64-linux-gnu
+	MAYBE_QEMU ?= qemu-aarch64 -L /usr/aarch64-linux-gnu
```

Comment:
Same goes for this change. If the user wants something else to run, with `?=` they can specify that without having to modify the makefile.